Source/Core/InputCommon/ControlReference/ExpressionParser.cpp

```diff
@@ -371,6 +369,7 @@ class BinaryExpression : public Expression
     }
     case TOK_ASSIGN:
     {
+      // You can do crazy things with this
```

Comment:
This comment seems a bit off, should it serve any particular purpose? It doesn't really explain _why_ you can do crazy things (or even more specifically, _what_ crazy things), so I don't see much point in adding it like that.

Reply:
Definitely agree, I'm not the type to write these comments and I knew someone would notice it, I will clarify it.